quickstarts/key-vault/src/secrets.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  SecretClient,
+  KeyVaultSecret,
+  SecretProperties,
+} from "@azure/keyvault-secrets";
+import { DefaultAzureCredential } from "@azure/identity";
+import "dotenv/config";
+
+// Passwordless credential
+const credential = new DefaultAzureCredential();
+
+// Get Key Vault name from environment variables
+const keyVaultName = process.env.KEY_VAULT_NAME;
+if (!keyVaultName) throw new Error("KEY_VAULT_NAME is empty");
+
+// URL to the Key Vault
+const url = `https://${keyVaultName}.vault.azure.net`;
+
+function printSecret(secret: KeyVaultSecret) {
+  const { name, value, properties } = secret;
+  const { enabled, expiresOn, createdOn } = properties;
+  console.log("Secret: ", { name, value, enabled, expiresOn, createdOn });
+}
+function printSecretProperties(secret: SecretProperties) {
+  const { name, enabled, expiresOn, createdOn } = secret;
+  console.log("Secret: ", { name, enabled, expiresOn, createdOn });
+}
+
+async function main() {
+  // Create a new SecretClient
+  const client = new SecretClient(url, credential);
+
+  // Create a unique secret name
+  const uniqueString = new Date().getTime().toString();
+  const secretName = `secret${uniqueString}`;
+
+  // Create a secret
+  const createSecretResult: KeyVaultSecret = await client.setSecret(
```

Comment:
These type annotations (they're in a few places) are redundant and can be removed. TypeScript is able to infer the type. I sort of understand if you want to demonstrate what type is being returned but IMHO it adds noise and isn't idiomatic. We don't add these annotations in the SDK samples.

```suggestion
  const createSecretResult = await client.setSecret(
```